<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="0" skipped="1">
  <reporter-output>
    <line>
      <![CDATA[lauching chromeBrowser]]>
    </line>
    <line>
      <![CDATA[Lauching website]]>
    </line>
    <line>
      <![CDATA[Waiting for 3000 milisec]]>
    </line>
    <line>
      <![CDATA[Waiting for 2000 milisec]]>
    </line>
    <line>
      <![CDATA[Clicking on roundTrip]]>
    </line>
    <line>
      <![CDATA[Waiting for 2000 milisec]]>
    </line>
    <line>
      <![CDATA[Waiting for 1000 milisec]]>
    </line>
    <line>
      <![CDATA[Entering from city name]]>
    </line>
  </reporter-output>
  <suite started-at="2023-10-25T09:51:32 IST" name="Default suite" finished-at="2023-10-25T09:55:10 IST" duration-ms="217441">
    <groups>
    </groups>
    <test started-at="2023-10-25T09:51:32 IST" name="Default test" finished-at="2023-10-25T09:55:10 IST" duration-ms="217441">
      <class name="test.TC01_verifyMessageOf30Days">
        <test-method is-config="true" signature="OpenBrowser()[pri:0, instance:test.TC01_verifyMessageOf30Days@1e0b4072]" started-at="2023-10-25T09:51:32 IST" name="OpenBrowser" finished-at="2023-10-25T09:54:43 IST" duration-ms="191012" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[lauching chromeBrowser]]>
            </line>
            <line>
              <![CDATA[Lauching website]]>
            </line>
          </reporter-output>
        </test-method> <!-- OpenBrowser -->
        <test-method is-config="true" signature="journeyInfo()[pri:0, instance:test.TC01_verifyMessageOf30Days@1e0b4072]" started-at="2023-10-25T09:54:44 IST" name="journeyInfo" finished-at="2023-10-25T09:55:10 IST" duration-ms="26199" status="FAIL">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[testdata\mytest.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: testdata\mytest.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at util.Utility.readExcelData(Utility.java:34)
at pom.HomePage.enterFromcityname(HomePage.java:92)
at test.TC01_verifyMessageOf30Days.journeyInfo(TC01_verifyMessageOf30Days.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:823)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:590)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
            <line>
              <![CDATA[Waiting for 3000 milisec]]>
            </line>
            <line>
              <![CDATA[Waiting for 2000 milisec]]>
            </line>
            <line>
              <![CDATA[Clicking on roundTrip]]>
            </line>
            <line>
              <![CDATA[Waiting for 2000 milisec]]>
            </line>
            <line>
              <![CDATA[Waiting for 1000 milisec]]>
            </line>
            <line>
              <![CDATA[Entering from city name]]>
            </line>
          </reporter-output>
        </test-method> <!-- journeyInfo -->
        <test-method signature="VerifyMessageOF30Days()[pri:0, instance:test.TC01_verifyMessageOf30Days@1e0b4072]" started-at="2023-10-25T09:55:10 IST" name="VerifyMessageOF30Days" finished-at="2023-10-25T09:55:10 IST" duration-ms="0" status="SKIP">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[testdata\mytest.xlsx (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: testdata\mytest.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at util.Utility.readExcelData(Utility.java:34)
at pom.HomePage.enterFromcityname(HomePage.java:92)
at test.TC01_verifyMessageOf30Days.journeyInfo(TC01_verifyMessageOf30Days.java:35)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:823)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:590)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- VerifyMessageOF30Days -->
      </class> <!-- test.TC01_verifyMessageOf30Days -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
